.github/workflows/add-issues-to-project.yaml

```diff
@@ -33,7 +33,9 @@ jobs:
           git clone --single-branch --depth 1 "https://github.com/kata-containers/.github" && cd .github/scripts
           sudo install hub-util.sh /usr/local/bin
           popd &>/dev/null
-
+      - name: chown file access
+        run: |
+          sudo chown -R $USER:$USER /home/github/CI/actions-runner/_work/kata-containers
```

Comment:
1. I think this can be prettified a little with `$GITHUB_WORKSPACE`.
2. Honestly, I wonder why GHA is attempting `git clean` at all. Aren't those environments immediately thrown away anyhow?
3. Is this necessary for all the workflows? E.g. I don't see how we're adding anything to the workspace by root in this workflow here.

Reply:
1. Thanks for the reminder, using $GITHUB_WORKSPACE to represent the path is more general.
2. This is the logic of GHA's CI yaml. Perhaps this can ensure that the test environment of each job is consistent with the upstream.
3. I refered to the configuration tutorial of "Add Github Action Runner" page, the CLI warns "Don't must use root privileges to start the runner", so my modifications here are all for non-root users.

Reply:
3. I'm really not in favour of `chown`ing in lots of places without necessity. Can you please limit this to the jobs where you saw issues with permissions?